src/bin/compute-wer.cc

```diff
@@ -128,7 +128,8 @@ int main(int argc, char *argv[]) {
       }
       num_words += ref_sent.size();
       int32 ins, del, sub;
```

Comment:
Oh... if this is only for debugging, can you just not make these changes
part of this PR?
Incidentally, there is a PR by @qbolec that impacts on this.  Ideally we
should favor substitutions over insertions+deletions, all things being
equal, as it gives a nicer-looking alignment, but the current code just
gives you an arbitrary alignment if there are ties in the Levenshtein
distance.
Dan


On Fri, May 27, 2016 at 5:49 PM, Vimal Manohar <notifications@github.com>
wrote:

> In src/bin/compute-wer.cc
> <https://github.com/kaldi-asr/kaldi/pull/777#discussion_r64970871>:
>
> > @@ -128,7 +128,8 @@ int main(int argc, char *argv[]) {
> >        }
> >        num_words += ref_sent.size();
> >        int32 ins, del, sub;
>
> I will avoid using bool as int.
> This change is only for debugging purposes. The script finally uses
> align-text and uses that output.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/777/files/bd6495b2eb866e8145005d64646728eefa023368..db82996f16f13cb9bdf38a4f8a0b628da19451a4#r64970871>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADJVu_pIAaxH0DVT8_CYXkL9y0J1ENS6ks5qF2blgaJpZM4IeUC0>
> .
>